frontend/src/components/eam-configurate-worker/eam-configurate-worker.tsx

```diff
@@ -112,6 +119,15 @@ const EAMConfigurateWorker: FC = () => {
             </li>
           </ul>
           <div className={styles.buttons}>
+            {hasCsvColumns && (
```

Comment:
```suggestion
<div className={styles.buttonContainer}>
            {hasCsvColumns && (
```
you need one, not two wrappers